### PR TITLE
Fix pv copy help

### PR DESCRIPTION
**Pull request by spluta**
Updated FFT Overview and PV_Copy help files to reflect the current syntax for automatically copying FFT data for parallel PV streams.

### CODE REVIEW

File: HelpSource/Guides/FFT-Overview.schelp [hunk 1]
```diff
@@ -80,28 +80,38 @@ d = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
 d.free;
 ::
 
-Because each PV UGen overwrites the output of the previous one, it is necessary to copy the data to an additional buffer at the desired point in the chain in order to do parallel processing of input without using multiple FFT UGens. link::Classes/PV_Copy:: allows for this.
+As seen below, SuperCollider can create chains of PV processes, which alter the same FFT buffer in series. The PV Ugen writes the output of its process in place, and the effect of one process feeds into the next:
+
 code::
 (
```
Comment (telephon) [inline HelpSource/Guides/FFT-Overview.schelp]: could you please inline using tabs? In general: documentation and sclang uses tabs, and the c++ side of sc uses spaces.

File: HelpSource/Guides/FFT-Overview.schelp [hunk 2]
```diff
@@ -80,28 +80,38 @@ d = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
...
+	[IFFT(chain0), IFFT(chain1)*0.1];
 }.play;
 )
-x.free;
-// IFFTed frames contain the same windowed output data
-b.plot(\b, Rect(200, 430, 700, 300), nil, nil); c.plot(\c, Rect(200, 100, 700, 300), nil, nil);
-[b, c].do(_.free);
 ::
 
+note::Parallel processing was previously acheived using PV_Copy. PV_Copy code should continue to work, but new code should be written as it appears above.::
```
Comment (telephon) [inline HelpSource/Guides/FFT-Overview.schelp]: achieved

File: HelpSource/Classes/PV_Copy.schelp [hunk 3]
```diff
@@ -6,7 +6,9 @@ description::
 
 Copies the spectral frame in bufferA to bufferB at that point in the chain of PV UGens. This allows for parallel processing of spectral data without the need for multiple FFT UGens, and to copy out data at that point in the chain for other purposes. bufferA and bufferB must be the same size.
 
-note::As of SC 3.7 instances of PV_Copy are added automatically where necessary for parallel processing. This document is provided for legacy purposes only. (Existing code explicitly using PV_Copy should continue to work.)::
+note::As of SC 3.7 instances of PV_Copy are added automatically where necessary for parallel processing. Please see link::Guides/FFT-Overview:: for the current implementation. 
+
+This document is provided for legacy purposes only. (Existing code explicitly using PV_Copy should continue to work.)::
```
Comment (LFSaw) [inline HelpSource/Classes/PV_Copy.schelp]: remove brackets?

**Comment by spluta** [comment]
I didn’t really edit the PV_Copy help file, because I was told that it was depreciated. But I would happily do that.

This is my first pull request. Do I go back into my branch and make these edits or is that done on the master? Do I need to resubmit the pull request after I make these edits?

Sam

**Comment by LFSaw** [comment]
Do you have that branch checked out on your local machine? Then you just edit the file, save, commit, and push the changes to the repo on your github account. Changes should then show up automatically in this PR.

as a side note, for your next PR instead of `fix-PVCopy-Help`, please add a `topic/` in front of your branch name: `topic/fix-PVCopy-Help`. this makes searching for the branch a lot easier.

**Comment by telephon** [comment]
ah I see now, it sums up to a big mix of things, so the current one is fine.

File: HelpSource/Classes/PV_Copy.schelp [hunk 4]
```diff
@@ -17,129 +17,53 @@ source buffer.
...
-	chain = PV_RectComb(chain, 20, 0, 0.2);
-	PV_Copy(chain, LocalBuf(2048)); // after comb
-	2.do({chain = PV_MagSquared(chain)});
-	PV_Copy(chain, LocalBuf(2048)); // after magsquared
-	0.00001 * Pan2.ar(IFFT(chain));
+	0.1 * IFFT([chain, chainB]);
 }.play(s);
 )
 x.free;
-
```
Comment (telephon) [inline HelpSource/Classes/PV_Copy.schelp]: is there a place where these examples still exist? They might be useful perhaps.

**Comment by spluta** [comment]
Which ones? The first few examples in the old PV_Copy help file were uniquely bad, in that they both didn’t clearly sonically portray what was going on AND they didn’t even require PV_Copy, thus being fairly misleading to the new user.

The plot stuff at the bottom could go back in, but in the current version, the code doesn’t work (it copies into nebulous LocalBufs instead of c, e, and f…unless I am misreading it. Should I fix it and put it back in? I really don’t think it clarifies what PV_Copy is.

Sam

**Comment by telephon** [comment]
They don't illustrate anything here, you are right.

But is there some place in the FFT helpfiles that shows how to plot?

Like 

```
(
c.getToFloatArray(action: { arg array;
    var z, x;
    z = array.clump(2).flop;
    // Initially data is in complex form
    z = [Signal.newFrom(z[0]), Signal.newFrom(z[1])];
    x = Complex(z[0], z[1]);
    { x.magnitude.plot('Initial', Rect(200, 560, 700, 200)) }.defer
});
```

**Comment by spluta** [comment]
I can put it in FFT Overview. Seems like the right place.

Sam

**Comment by telephon** [comment]
good, thanks!

**Comment by spluta** [comment]
OK. I think this is much better. I updated FFT Overview to have a section on Accessing Frame Data for the plot stuff and then added a section marker for PV_ChainUGens and a link to that help file.

Sam

**Comment by telephon** [comment]
Is this ready to merge or do you want to add more fixes?

**Comment by spluta** [comment]
I’m done.

Sam

**Pull request merged**